logging/tests/unit/handlers/transports/test_background_thread.py

```diff
@@ -29,36 +29,41 @@ def _get_target_class():
         return BackgroundThreadTransport
 
     def _make_one(self, *args, **kw):
-        return self._get_target_class()(*args, **kw)
+        worker_patch = mock.patch(
+            'google.cloud.logging.handlers.transports.'
+            'background_thread._Worker',
+            autospec=True)
+        with worker_patch as worker_mock:
+            return self._get_target_class()(*args, **kw), worker_mock
 
     def test_ctor(self):
         client = _Client(self.PROJECT)
-        NAME = 'python_logger'
-        transport = self._make_one(client, NAME)
-        self.assertEqual(transport.worker.logger.name, NAME)
+        name = 'python_logger'
+
+        transport, worker = self._make_one(client, name)
+
+        logger = worker.call_args[0][0]
```

Comment:
Can you be more explicit? Like check `len(worker.call_args)` and `len(worker.call_args[0])`? (i.e. on reading this I have no idea what you **aren't** checking)

ALSO: Feel free to rename this `test_constructor`